### PR TITLE
more navbar changes to be responsive

### PR DESCRIPTION
**Pull request by doowee0509**
Put all of the icon function inside a dropdown to save space on mobile devices. 
Category dropdown can now open on div click and not just the dropdown arrow.
Changed font size and hero banner size for mobile look.

### CODE REVIEW

File: capstone-ui/src/Components/RecipeDetail/RecipeDetail.jsx [hunk 1]
```diff
@@ -307,8 +331,13 @@ function RecipeReview({recipeId}) {
         </div>
         <div className='add-review-text'>
           {/* Conditional rendering for when the user is logged in/ not logged in */}
-          {user?.email ? <TextareaAutosize placeholder={'Leave a review'} onChange={handleOnInputChange} value={comment} style={{ minHeight: 20}}/>
-          : <span>You must be <span className="links" onClick={showLoginForm}>logged in</span> to leave a review. Don't have an account? Sign up <span className="links" onClick={showRegisterForm}>here!</span></span>
+          {console.log(window.innerWidth) > 420
```
Comment (Rebecca-CYLi) [inline capstone-ui/src/Components/RecipeDetail/RecipeDetail.jsx:334]: found a console log

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
found a console log, please fix it

File: capstone-ui/src/Components/RecipeDetail/RecipeDetail.jsx (hunk 1 above)
Comment (doowee0509) [reply capstone-ui/src/Components/RecipeDetail/RecipeDetail.jsx]: Console log removed

**Review by Rebecca-CYLi — APPROVED** [review]
revision made, LGTM

**Pull request merged**